### ISSUE TITLE
Masking for two population pipeline

### ISSUE THREAD

**Issue by ckyriazis** [popgensims/analysis#43]
Wanted to get some input on how the masking should be done for the two population pipeline, since it seems like it will be a bit different from n_t where the masks are just given to each method. 

Does it make sense to just mask the tree sequences after the simulation is complete and before generating the SFS? Does tskit have a mask function? @andrewkern @jeromekelleher

**Comment by andrewkern** [comment]
Yes that is the way I've been doing it for the SFS in the one population setting. Check out this bit of code for going from `tree_sequence` to `stairwayplot` input using masking
https://github.com/popgensims/analysis/blob/607772593ef8042a18d72dea2098e4d55f87dcc0/n_t/stairway.py#L47

you should be able to use that same exact logic to get the masked SFS for input to dadi, etc.

**Comment by ckyriazis** [comment]
Got it - thanks!

**Comment by jeromekelleher** [comment]
Looks like this is sorted - can we close?

**Comment by andrewkern** [comment]
i think so. @ckyriazis i'm closing this down. feel free to reopen if need be.

**Comment by ckyriazis** [comment]
Opening this again since I ran into an issue trying to implement this in the two population pipeline, which replicated when I tried running the n_t pipeline. 

Using the n_t pipeline and the default config.json on the readme, I'm getting a numpy "too many indices for array" error thats pointing to this line from the ts_to_stairway function: 

`allele_counts = allel.HaplotypeArray(haps[retain, :]).count_alleles()`

The same error is causing me problems when I try to use a similar approach in the two_population pipeline. Any idea whats going wrong for me here?
